container/Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM registry.access.redhat.com/ubi7/ubi:latest
 LABEL maintainer "Chen Gao <Chen.Gao@.ibm.com>"
 
-ENV IOT_REPO_PATH=/root/db2eventstore-IoT-Analytics \
-    KAFKA_REPO_PATH=/root/db2eventstore-kafka
-ARG SETUP_AREA=${IOT_REPO_PATH}/container/setup
+ENV IOT_REPO_PATH=/root/db2eventstore-IoT-Analytics
+ENV KAFKA_REPO_PATH=/root/db2eventstore-kafka 
+ENV SPARK_HOME="/spark_home"
+ENV PATH=$PATH:$SPARK_HOME/bin:$SPARK_HOME/sbin
+ARG ES_VERSION=latest
 ARG BRANCH=master
+ARG SETUP_AREA=${IOT_REPO_PATH}/container/setup
```

Comment:
Why is `SETUP_AREA` and `ARG` ?  I don't believe it's passed in by anyone ?  i.e. should it not be an `ENV` like the rest above ?  

`BRANCH` and `ES_VERSION` make sense as those are args (with the default values above) passed in by `build.sh`.

Reply:
setup area was defined as ARG in the original script so I did not change it